### PR TITLE
Resource Graph: updated package Microsoft.Azure.Management.ResourceGraph to version 2.0

### PR DESCRIPTION
**Pull request by OlhaTkachenko**
Updated package Microsoft.Azure.Management.ResourceGraph to version 2.0 and utilizing new ResultFormat objectArray from the package

## Checklist

- [x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#updating-the-change-log)
- [x] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (Microsoft internal partners only)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by adxsdkps** [comment]
Can one of the admins verify this patch?

File: src/ResourceGraph/ResourceGraph/Utilities/ResultExtensions.cs [hunk 1]
```diff
@@ -49,41 +42,63 @@ public static class TableExtensions
...
-                column.Type == IdColumnType);
+            try
+            {
+                if (data is JArray array)
+                {
+                    return ToPsObjects(array.ToList());
+                }
 
-            foreach (var row in table.Rows)
+                throw new ArgumentOutOfRangeException("Result format is not supported");
```
Comment (demyanenko) [inline src/ResourceGraph/ResourceGraph/Utilities/ResultExtensions.cs]: This exception will be double thrown. Can we optimize?

**Comment by cormacpayne** [comment]
@OlhaTkachenko the change to update the version of `Microsoft.Rest.ClientRuntime` has been merged, so you will need to pull the latest changes from `master` into your branch.

File: src/ResourceGraph/ResourceGraph/Utilities/ResultExtensions.cs (hunk 1 above)
Comment (OlhaTkachenko) [reply src/ResourceGraph/ResourceGraph/Utilities/ResultExtensions.cs]: Done

**Comment by OlhaTkachenko** [comment]
Fixing tests

**Pull request merged**